prow/cmd/build/controller.go

```diff
@@ -57,6 +59,19 @@ const (
 	controllerName = "prow-build-crd"
 )
 
+type pjClient struct {
+	pjc prowjobsetv1.ProwJobInterface
+}
+
+func (c *pjClient) ReplaceProwJob(name string, pj prowjobv1.ProwJob) (prowjobv1.ProwJob, error) {
+	npj, err := c.pjc.Update(&pj)
+	if npj != nil {
+		return *npj, err
+	} else {
```

Comment:
else is not necessary here

https://golang.org/doc/effective_go.html#if

Reply:
removed else